src/apps/relay/dbdrivers/dbdriver.c

```diff
@@ -111,3 +111,39 @@ const turn_dbdriver_t * get_dbdriver()
 	return _driver;
 }
 
+char* sanitize_userdb_string(char* udb) {
+    char * ret = NULL;
+    char * pstart;
+    char * pend;
+
+    /* host=localhost dbname=coturn user=turn password=turn connect_timeout=30 */
+    ret = strdup(udb);
+    pstart = strstr (ret,"password=");
+    if (pstart != NULL)
+    {
+        pstart += strlen("password=");
+        pend = strstr (pstart," ");
+        size_t plen = pend - pstart;
+        if (pend == NULL)
+        {
+            plen = strlen(pstart);
+        }
+        memset(pstart,'*',plen);
+    }
+    else
+    {
+        /* postgresql://username:password@/databasename */
+        pstart = strstr (ret,"postgresql://");
```

Comment:
username is being replaced with '*' in this case.   I guess it is a known issue but just in case.    it lgtm anyway

Reply:
> I think it is missing the same change in dbd_redis.c.
> 
> Rest of comments are not critical, just fyc if they make sense.
My editor decided to skip the code text that is not enabled due to ifdef ...